docs/PythonAPIOverview.md

```diff
@@ -280,27 +293,27 @@ In this example we merge two models by connecting each output of the first model
 ```python
 import onnx
 
-model1 = onnx.load('path/to/model1.onnx')
+model1 = onnx.load("path/to/model1.onnx")
 # agraph (float[N] A, float[N] B) => (float[N] C, float[N] D)
 #   {
 #      C = Add(A, B)
 #      D = Sub(A, B)
 #   }
 
-model2 = onnx.load('path/to/model2.onnx')
+model2 = onnx.load("path/to/model2.onnx")
 #   agraph (float[N] X, float[N] Y) => (float[N] Z)
 #   {
 #      Z = Mul(X, Y)
 #   }
 
 combined_model = onnx.compose.merge_models(
     model1, model2,
-    io_map=[('C', 'X'), ('D', 'Y')]
+    io_map=[("C", "X"), ("D", "Y")]
```

Comment:
I would have one this change in a separate PR. It would be easier to read.

Reply:
Sounds good. I have create a specific PR for it: https://github.com/onnx/onnx/pull/4530.